platform/net/ubuntu_net_manager.go

```diff
@@ -89,20 +89,24 @@ func (net UbuntuNetManager) SetupIPv6(config boshsettings.IPv6, stopCh <-chan st
 
 func (net UbuntuNetManager) SetupNetworking(networks boshsettings.Networks, mbus string, errCh chan error) error {
 	if networks.IsPreconfigured() {
-		err := net.setupFirewall(mbus)
+
+		// Note in this case IPs are not broadcast
+		dnsNetwork, _ := networks.DefaultNetworkFor("dns")
+		net.logger.Info(UbuntuNetManagerLogTag, "Successfully set up outgoing nats api firewall")
```

Comment:
Is this log message really correct? The firewall rules are not set up at this point.